### ISSUE TITLE
Defining Overlapping Policies for Distinct Activity Types

### ISSUE THREAD

**Issue by lummish** [openmobilityfoundation/curb-data-specification#138]
### Is your feature request related to a problem? Please describe.

We are having trouble defining a policy which overlaps in time and user classes with another policy, but specifies regulations for a different activity type.

For a representative example, consider the following situation:

Regulation A: There is a loading zone which has normal operating hours between 09:00 and 17:00. In this zone, we allow all types of vehicle to park during the specified hours.

Regulation B: We would like to define a 24-hour policy for this zone which specifies that vehicles cannot stop in the travel lane along the zone (i.e. vehicles are not allowed to double park). This regulation applies to all classes of vehicle.

Currently, it does not seem possible to specify both Regulation A and Regulation B as separate policies because, though the policies specify regulations regarding different activities (`parking` vs. `travel`), they do not do so using disjoint user classes.

### Describe the solution you'd like

Modify the specification to allow policies with overlapping time spans to be defined with equal priorities on the same zone if they specify disjoint user classes _or_ disjoint activities. 

### Is this a breaking change

The only potential breaking change that this proposal would introduce would be with respect to policy validation logic, as policies which would previously have been considered as conflicting, would not necessarily be conflicting.

### Impacted Spec

For which spec is this feature being requested?

 * `Curbs`

### Describe alternatives you've considered

- We considered defining the travel lane as a separate zone which would allow specification of a separate policy, however a standard travel lane does not meet the criteria specified for being a zone.
- We considered allowing conflicting activity definitions for rules within an individual policy, however this does not accommodate regulations which have distinct time spans.

cc: @michaelautomotus

**Comment by schnuerle** [comment]
@lummish Can you recommend a specific change to the spec that would accommodate what you are asking for? Is it enough to just allow equal policy priorities?

**Comment by lummish** [comment]
@schnuerle I don't believe any material change to the specification would be required other than modifying the provision in the "priority" section [here](https://github.com/openmobilityfoundation/curb-data-specification/blob/main/curbs/README.md#policy). Currently, it reads:

> Two Policies that apply to the same Curb Zone with overlapping Time Spans and equivalent User Class enumerations MUST NOT have the same priority.

This could be changed to:

> Two Policies that apply to the same Curb Zone with overlapping Time Spans MAY ONLY have the same priority if the policies apply to disjoint User Classes or disjoint Activities.

**Comment by LaurentG-AMD** [comment]
Is there really a conflict if both activities are disjointed? If car A is performing activity A covered by policy A, isn't policy B that does not cover it just irrelevant? So setting whichever priority higher than the other achieve the same result anyways?

**Comment by jacobmalleau** [comment]
Also, would this not be considered two separate curb zones? There is the curb adjacent zone which has the loading policy (Policy A), and then the travel lane with the no stopping policy (Policy B).

 I know technically CDS does not cover travel lanes, but could in this context.

**Comment by schnuerle** [comment]
We certainly could change the spec to allow the use of CDS in travel lanes. I think this is something we have talked about.

**Comment by schnuerle** [comment]
I think we can modify this language for 1.1 without it being breaking. Is this still a need for you, vs just having a different priority for different policies?

Currently, it reads:

> Two Policies that apply to the same Curb Zone with overlapping Time Spans and equivalent User Class enumerations MUST NOT have the same priority.

This could be changed to:

> Two Policies that apply to the same Curb Zone with overlapping Time Spans MAY ONLY have the same priority if the policies apply to disjoint User Classes or disjoint Activities.

**Comment by schnuerle** [comment]
Loosened the policy priority rules with this [commit](https://github.com/openmobilityfoundation/curb-data-specification/pull/194/commits/6d7a2d74f58124d5552f2391df4b1a38c2eeb0a2) in #194.
